.github/workflows/build-matter-telink-b92.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Matter for Telink B92
+
+on:
+  push:
+    branches:
+      - main
+      - setup-github-actions
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      GCC_PATH: ${{ github.workspace }}/toolchain/gcc-arm-none-eabi-10-2020-q4-major/bin
+      PATH: $GCC_PATH:$PATH
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Cache Python dependencies
+      uses: actions/cache@v3
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
```

Comment:
this seems to be a high time-cost for every build.

Can we please have a docker image instead?